### PR TITLE
Internal: fix install gsutil issues on suse distros

### PR DESCRIPTION
**Pull request by Subbarker**
## Related issue

b/337015263

## Checklist:
- Unit tests
  - [x] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [ ] Integration tests do not apply.
  - [x] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [x] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [ ] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [ ] This PR bumps the version.

### CODE REVIEW

File: integration_test/gce/gce_testing.go [hunk 1]
```diff
@@ -1550,8 +1550,8 @@ func InstallGsutilIfNeeded(ctx context.Context, logger *log.Logger, vm *VM) erro
 
 	// SUSE seems to be the only distro without gsutil, so what follows is all
 	// very SUSE-specific.
-	if !IsSUSE(vm.ImageSpec) {
-		return installErr("gsutil", vm.ImageSpec)
+	if vm.OS.ID != "opensuse" && vm.OS.ID != "sles" && vm.OS.ID != "sles_sap" {
```
Comment (jefferbrecht) [inline integration_test/gce/gce_testing.go:1553]: Nit: factor this (and the other check from line 1265) to a function?

**Review by jefferbrecht — APPROVED** [review]

File: integration_test/gce/gce_testing.go (hunk 1 above)
Comment (Subbarker) [reply integration_test/gce/gce_testing.go:1553]: I've got a followup for that.

Comment (igorpeshansky) [inline integration_test/gce/gce_testing.go:1553]: Why are we not changing `IsSUSE` to use `vm.OS.ID` and then calling that?

Comment (franciscovalentecastro) [reply integration_test/gce/gce_testing.go:1553]: This was the approach we ended up using in https://github.com/GoogleCloudPlatform/ops-agent/pull/1720 .

**Pull request merged**